Root/OptimizationDump.cxx

```diff
@@ -35,7 +35,11 @@
 namespace HF = HelperFunctions;
 namespace VD = VariableDefinitions;
 
-#define ARRAY_INIT {-99, -99, -99, -99}
+
+#define N_LARGE_R 8
```

Comment:
My attempt to make this general. Should probaly go in the header and be used there too though.